areas/acr/tests/AzureMcp.Acr.LiveTests/AcrCommandTests.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using AzureMcp.Core.Models;
+using AzureMcp.Tests;
+using AzureMcp.Tests.Client;
+using AzureMcp.Tests.Client.Helpers;
+using Xunit;
+
+namespace AzureMcp.Acr.LiveTests;
+
+[Trait("Area", "Acr")]
+[Trait("Category", "Live")]
+public class AcrCommandTests(LiveTestFixture liveTestFixture, ITestOutputHelper output)
+    : CommandTestsBase(liveTestFixture, output), IClassFixture<LiveTestFixture>
+{
+    [Theory]
+    [InlineData(AuthMethod.Credential)]
+    public async Task Should_ListRegistries_Successfully(AuthMethod authMethod)
+    {
+        // Arrange & Act
+        var result = await CallToolAsync(
+            "azmcp_acr_registry_list",
+            new()
+            {
+                { "subscription", Settings.SubscriptionId },
+                { "auth-method", authMethod.ToString() }
+            });
+
+        // Assert
+        var registries = result.AssertProperty("registries");
+        Assert.Equal(JsonValueKind.Array, registries.ValueKind);
+        // Note: We don't assert specific registries exist since test environment may not have any
```

Comment:
Shouldn't we then set up the test environment to have at least one and make sure we can retrieve it?